src/Dropdown.php

```diff
@@ -2691,7 +2691,13 @@ public static function getDropdownValue($post, $json = true)
             if (isset($post['condition']['WHERE'])) {
                 $where = array_merge($where, $post['condition']['WHERE']);
             } else {
-                $where = array_merge($where, $post['condition']);
+                foreach ($post['condition'] as $key => $value) {
+                    if (strpos($key, '.') === false) {
+                        $where["$table.$key"] = $value;
```

Comment:
```suggestion
                        // Ensure condition contains table name to prevent ambiguity with fields from `glpi_entities` table
                        $where["$table.$key"] = $value;
```